AndroidXCI/lib/src/main/kotlin/dev/androidx/ci/testRunner/TestRunnerServiceImpl.kt

```diff
@@ -261,7 +262,7 @@ internal class TestRunnerServiceImpl internal constructor(
                     }.add(
                         TestRunnerService.TestCaseArtifact(
                             ResultFileResourceImpl(visitor),
-                            visitor.fileName.substringAfterLast(".")
+                            visitor.fileName.substringAfterLast(".").uppercase(Locale.getDefault())
```

Comment:
we should probably use US english locale here as thats what we'd normally expect.

Reply:
Done